src/Components/Artifacts/Hero/Artifacts.module.css

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
Well done! bara ta bort blankrader 1, 2 o 12 i CSSen

Reply:
Thank you! Har tagit bort blankraderna samt lagt i bokstavordning :)